pipeline/transformer/transformer_calm/src/main/scala/uk/ac/wellcome/platform/transformer/calm/CalmTransformer.scala

```diff
@@ -37,12 +38,10 @@ object CalmTransformer
   def apply(record: CalmRecord, version: Int): Result[Work[Source]] =
     if (shouldSuppress(record)) {
       Right(
-        Work.Invisible[Source](
+        Work.Deleted[Source](
```

Comment:
I'm not sure this is quite right – a work can be suppressed but still present in the source system, right? Possibly I'm splitting hairs, but I'd prefer to keep suppressed-but-extant works as Invisible.

Reply:
Maybe it's a naming issue. The main drive for splitting `Deleted` out of `Invisible` is that `Invisible` is used in the METS transformer (and Sierra I think) to send through works that are incomplete, so shouldn't be shown on their own. METS works are always considered incomplete and sent as `Invisible`. However, the merger can use the information in `Invisible` works to merge into `Visible` works. We don't want that for `Deleted` works. So the discrimination in my mind has been: is what we have in this work usable in the pipeline to enrich other works? yes => make it `Invisible`, no => make it "Deleted". I think `Suppressed` falls into the second case?

Reply:
If I was really splitting hairs, I'd introduce _another_ work state, so we'd have Invisible/Suppressed/Deleted, but I don't think it's worth holding up this PR.